sdk/eventhubs/azure_messaging_eventhubs/rust-toolchain.toml

```diff
@@ -1,2 +0,0 @@
-[toolchain]
-channel = "1.78.0"
```

Comment:
Did you have a specific need for 1.78? Our current MSRV is 1.76.

Reply:
I did - that's the version that was used for iron oxide. But it works just fine with 1.76 so I removed the file (it was accidentally added).